### ISSUE TITLE
django-inspectional-registration and custom_user

### ISSUE THREAD

**Issue by stushurik** [lambdalisue/django-inspectional-registration#48]
Hi.

I am using django-inspectional-registration with django 1.8 (previously 1.7). It seems that django-inspectional-registration uses south which is incompatible out of the box with django migrations.

The problem is that I am user custom user model. The django-inspectional-registration become "unmigrated app" and thus django tries create  django-inspectional-registration before my user model.

Note that I have tried to apply migration onto new database

**Comment by stushurik** [comment]
For now I use [MIGRATION_MODULES](https://docs.djangoproject.com/en/1.8/ref/settings/#std:setting-MIGRATION_MODULES) for registration app migrations and creating initial migration for this app

**Comment by lambdalisue** [comment]
**First of all, we are using django-inspectional-registration in Django 1.8 thus I don't really think it is django-inspectional-registration problem.**

> It seems that django-inspectional-registration uses south which is incompatible out of the box with django migrations.

You can upgrade from south. So it is not really incompatible.
https://docs.djangoproject.com/en/1.8/topics/migrations/#upgrading-from-south

I don't really get the point of this issue... It seems "The django-inspectional-registration become "unmigrated app" and thus django tries create django-inspectional-registration before my user model." is a issue but I don't really know such behaviour. So could you tell me

* Problems summary
* Procedure
* Expect behaviour
* Actual behaviour

To make it clear for me?

**Comment by stushurik** [comment]
* Problems summary:

I have a cutom user app, with migrations. I have placed custom_user app before registration app in the INSTALLED_APPS. When I launch manage.py migrate for the first time (the DB doesn`t have any table), I can see in console:
```shell
Operations to perform:
  Synchronize unmigrated apps: registration
  Apply all migrations: admin, custom_users
Synchronizing apps without migrations:
  Creating tables...
    Creating table registration
```
When it tries to create registration that has reference to the user model, there is no custom_user table. And it fails.

* Procedure
manage.py migrate

* Expect behavior
If I placed custom_user app before the registarion app in the INSTALLED_APPS than custom_user tables would be created earlier than registration app`s tables.

* Actual behavior
The registration tables are created first.

I think it is caused by that fact, that django-inspectional-registration uses south migrations
